tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/WeakHashing/WeakHashingTests.cs

```diff
@@ -70,7 +73,10 @@ await VerifyHelper.VerifySpans(spans, settings)
     [Trait("Category", "EndToEnd")]
     [Trait("RunOnWindows", "True")]
     [InlineData("DD_IAST_ENABLED", "false")]
-    [InlineData("DD_IAST_WEAK_HASH_ALGORITHMS", "")]
+    // if we define a variable as string.empty, in net462 Environment.GetEnvironmentVariable(key) returns null, which is equivalent to variable not defined.
+    // The vulnerabilities disabling will be implemented in the future. Altough setting DD_IAST_WEAK_HASH_ALGORITHMS to empty actually disables
+    // weak hashing, it is ok for testing, but it is not the recommended way to disable it for clients
```

Comment:
FYI, as came up recently with @anna-git, using `string.Empty` as a way of disabling features is a problematic approach, as it's impossible to set a `string.Empty` env var in some situations (e.g. Windows in IIS). Which means the feature cannot be disabled.

Reply:
Since the disabling of weakhash by setting the var to an empty string, I think that it would be better to test this by setting a nonexisting algorithm. Done!